src/index.js

```diff
@@ -377,26 +376,46 @@ function createConfig(options, entry, format, writeMeta) {
 							},
 						}),
 					!useTypescript && flow({ all: true, pretty: true }),
-					// Only used for async await
-					babel({
-						// We mainly use bublé to transpile JS and only use babel to
-						// transpile down `async/await`. To prevent conflicts with user
-						// supplied configurations we set this option to false. Note
-						// that we never supported using custom babel configs anyway.
-						babelrc: false,
-						exclude: 'node_modules/**',
-						plugins: [
-							require.resolve('@babel/plugin-syntax-jsx'),
-							[
-								require.resolve('babel-plugin-transform-async-to-promises'),
-								{ inlineHelpers: true, externalHelpers: true },
-							],
-							[
-								require.resolve('@babel/plugin-proposal-class-properties'),
-								{ loose: true },
-							],
-						],
-					}),
+					babel(
+						(await isFile('.babelrc')) || (await isFile('.babelrc.js'))
```

Comment:
after a thought - we should make our internal babel config a public `babel-preset-microbundle`, but should be done in a followup review - because it will require setting up a monorepo.

Reply:
ohh I like that idea of a dedicated `microbundle` preset 👍

Reply:
@Andarist Could we expose something like https://github.com/babel/babel-loader/#customized-loader from `rollup-plugin-babel`? I think it is very bad to encourage frameworks to implement their own resolution for `.babelrc` files and such, since it doesn't necessarily conform to Babel's own implementation details.

Reply:
> @Andarist Could we expose something like https://github.com/babel/babel-loader/#customized-loader from rollup-plugin-babel?

Sure, gonna look through this to see what it actually is & how it can be used to achieve goals here. 

> I think it is very bad to encourage frameworks to implement their own resolution for .babelrc files and such, since it doesn't necessarily conform to Babel's own implementation details.

Totally agreed - I was only pointing to babel-jest & others to see how similar goal is achieved. It's 100% better to reuse the logic in babel itself to avoid buggy implementations.